analysis/webservice/webapp.py

```diff
@@ -156,6 +155,23 @@ def async_callback(self, result):
         if hasattr(result, 'cleanup'):
             result.cleanup()
 
+def inject_args_in_config(args, config):
```

Comment:
Can you write a unit test for this function? There is some parsing logic which should be unit tested

Reply:
I wrote some unit test. I did some with mock before with java, I made my first mock in python.
I can not say I had more fun with it 😄